tests/features/news.feature

```diff
@@ -55,3 +55,41 @@ Feature: News content creation
     And I press "Save"
     # We assert that the extra characters are actually truncated from the end of the string.
     Then I should not see "The text to remove."
+
+  @purl-linkit @javascript
+  Scenario: Make sure that persistent url work with linkit module.
+    Given I am logged in as a user with the "create oe_news content, access content, edit own oe_page content, view published skos concept entities, create av_portal_photo media, use text format base_html, view the administration theme" permission
+    And the following languages are available:
+      | languages |
+      | en        |
+      | fr        |
+    When I visit "the News creation page"
+    And I fill in "Title" with "News1"
+    And I enter "Teaser text" in the "Teaser" wysiwyg editor
+    And I enter "Summary text" in the "Summary" wysiwyg editor
+    And I enter "Body text" in the "Body" wysiwyg editor
+    And I fill in "Subject" with "financing"
+    And I fill in "Author" with "European Patent Office"
+    And I fill in "Content owner" with "Committee on Agriculture and Rural Development"
+    And I press "Save"
+    Then I should see "News1"
+    #Create second node
+    When I visit "the News creation page"
+    And I fill in "Title" with "News2"
+    And I enter "Teaser text" in the "Teaser" wysiwyg editor
+    And I enter "Summary text" in the "Summary" wysiwyg editor
+    And I insert link to "News1" through wysiwyg editor in "Body" field
+    And I fill in "Subject" with "financing"
+    And I fill in "Author" with "European Patent Office"
+    And I fill in "Content owner" with "Committee on Agriculture and Rural Development"
+    And I press "Save"
+    Then I should see "News2"
+    # Check link to first node
+    And I should see link to "News1" node
```

Comment:
For this step, reuse the method `assertLinkWithHref()` once you got the URL of the node itself.